pkg/oc/cli/admin/mustgather/mustgather.go

```diff
@@ -142,24 +146,30 @@ func (o *MustGatherOptions) Run(rsyncCmd *cobra.Command) error {
 	if err != nil {
 		return err
 	}
+	o.PrinterCreated.PrintObj(ns, o.Out)
 	if !o.Keep {
 		defer func() {
-			err = o.Client.CoreV1().Namespaces().Delete(ns.Name, nil)
+			if err := o.Client.CoreV1().Namespaces().Delete(ns.Name, nil); err != nil {
+				fmt.Printf("%v", err)
```

Comment:
On monday, use `Fprintf` to direct to the error stream and put a newline on it, but this better than where we are today and in the hopes of landing for a beta build I'll merge.